<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alarmWindow</class>
 <widget class="QMainWindow" name="alarmWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/images/background.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="line_hour">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>160</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:white;
padding-bottom:7px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_minute">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>270</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:white;
padding-bottom:7px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_hour">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>110</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Hour&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_minute">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>220</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Minute&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_alarm">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>400</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#comboBox{
	border: 1px solid #ced4da;
	border-radius: 4px;
	padding-left:10px;	
}
#combBox::drop-down{
	border:0px;
}
#comboBox::on{
	border:4px solid #c2dbfe;
}
#comboBox QListView{
	border :1px solid rgba(0,0,0,10%);
	padding:5px;
	background-color:#fff;
	outline: 0px;
}
#comboBox QListView::item{
	padding-left: 10px;
	backgroud-color:#fff;
}
#comboBox QListView::item:hover{
	background-color:#1e90ff;
}
#comboBox QListView::item:selected{
	background-color:#1e90ff;
}
</string>
     </property>
     <item>
      <property name="text">
       <string>music1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>music2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>music3</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn_set">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>330</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_set{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#btn_set:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 226));
}
QPushButton#btn_set:pressed{	
	padding-left:3px;
	padding-top:3px;
	background-color:rgba(105, 118, 132, 200);
}</string>
     </property>
     <property name="text">
      <string>Set Clock</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>550</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_home{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#btn_home:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 226));
}
QPushButton#btn_home:pressed{	
	padding-left:3px;
	padding-top:3px;
	background-color:rgba(105, 118, 132, 200);
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
